### PR TITLE
Feat customised updatedb

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: elife/config/etc-updatedb.conf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PRUNE_BIND_MOUNTS="yes"
+# PRUNENAMES=".git .bzr .hg .svn"
+PRUNEPATHS="/tmp /var/spool /media /home/.ecryptfs /ext"
+PRUNEFS="NFS nfs nfs4 rpc_pipefs afs binfmt_misc proc smbfs autofs iso9660 ncpfs coda devpts ftpfs devfs mfs shfs sysfs cifs lustre tmpfs usbfs udf fuse.glusterfs fuse.sshfs curlftpfs ecryptfs fusesmb devtmpfs"
```
Comment (giorgiosironi) [inline elife/config/etc-updatedb.conf]: because these filesystems are slow or shared, I guess

File: elife/config/etc-updatedb.conf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+PRUNE_BIND_MOUNTS="yes"
```
Comment (giorgiosironi) [inline elife/config/etc-updatedb.conf]: ok, I don't think we use bind mounts

File: elife/config/etc-updatedb.conf [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+# PRUNENAMES=".git .bzr .hg .svn"
+PRUNEPATHS="/tmp /var/spool /media /home/.ecryptfs /ext"
```
Comment (giorgiosironi) [inline elife/config/etc-updatedb.conf]: do we use encryption anywhere? Well, no harm

File: elife/config/etc-updatedb.conf (hunk 1 above)
Comment (lsh-0) [reply elife/config/etc-updatedb.conf]: or temporary or network shares

File: elife/config/etc-updatedb.conf (hunk 3 above)
Comment (lsh-0) [reply elife/config/etc-updatedb.conf]: no, but I think builder-private is a real concern. I think new version of Salt allows you to mount an encrypted pillar on the master and target values within it

**Comment by lsh-0** [comment]
that was the default config that came with ubuntu 14.04. I then did another commit with our elife-specific change `/ext` so it's clear what I changed in the blame history

**Pull request merged**